tests/album-update.test.js

```diff
@@ -0,0 +1,49 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Update Album', async () => {
+  let artistId, albumId;
+
+  beforeEach(async () => {
+    await db.query('DELETE FROM Albums');
+    await db.query('DELETE FROM Artists');
+    const { rows } = await db.query(
+      'INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *',
+      ['Britney Spears', 'Pop']
+    );
+    artistId = rows[0].id;
+  });
+
+  const { rows } = await db.query(
+    'INSERT INTO Albums (name, year, artist_id) VALUES($1, $2, $3) RETURNING *',
+    ['...Baby One More Time', 2005, artistId]

```

Comment:
You can see here it doesnt work as the year here is 2005, then later you send 1999, and the expect is 199 and ti doesnt throw an error which means the test is not working